src/validators/update.ts

```diff
@@ -50,3 +54,52 @@ export const checkVinValidity = (currentVin: string, newVin: (string | undefined
   }
   return false;
 };
+
+export const validateUpdateVrmRequest = (event: APIGatewayProxyEvent) => {
+  const isPathInvalid = validateSysNumTimestampPathParams(event);
+  if (isPathInvalid) {
+    return isPathInvalid;
+  }
+  if (!event.body || !Object.keys(event.body).length) {
+    return {
+      statusCode: 400,
+      body: 'invalid request',
+    };
+  }
+
+  if (!event.headers.Authorization) {
+    return {
+      statusCode: 400,
+      body: 'Missing authorization header',
+    };
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const newVrm: string = JSON.parse(event.body).newVrm as string;
+
+  if (!newVrm) {
+    return {
+      statusCode: 400,
+      body: 'You must provide a new VRM',
+    };
+  }
+  return false;
+};
+
+// eslint-disable-next-line consistent-return
+export const validateVrm = (currentRecord: TechrecordGet, newVrm: string) => {
```

Comment:
might be worth expanding this to check the length and that the VRM contains no special characters? I can't see those checks happening else where